src/translations/main_nl.ts

```diff
@@ -421,47 +421,47 @@ p, li { white-space: pre-wrap; }
     </message>
     <message>
         <source>opaque when new mail is present,</source>
-        <translation type="unfinished"></translation>
+        <translation>ondoorzichtig als er nieuwe e-mails zijn;</translation>
```

Comment:
```suggestion
        <translation>ondoorzichtig als er nieuwe e-mails zijn,</translation>
```
I'm unsure here, did you want the semicolon or was it accidental?

Reply:
I thought the string below it followed this one, so I read the sentence out loud to determine if a comma or semicolon would sound better and decided to go with the semicolon. So it wasn't accidental.